site/server/siteBaking.tsx

```diff
@@ -333,70 +331,55 @@ export async function feedbackPage() {
     return renderToHtmlPage(<FeedbackPage />)
 }
 
-let cachedCovidCountryProfilePostFormatted: FormattedPost | undefined
-let cachedCovidCountryProfilePostFormattingOptions:
-    | FormattingOptions
-    | undefined
-async function getCovidCountryProfilePost(
-    grapherExports?: GrapherExports
-): Promise<[FormattedPost, FormattingOptions]> {
-    if (
-        cachedCovidCountryProfilePostFormatted &&
-        cachedCovidCountryProfilePostFormattingOptions
-    ) {
-        return [
-            cachedCovidCountryProfilePostFormatted,
-            cachedCovidCountryProfilePostFormattingOptions
-        ]
-    }
-
-    // Get formatted content from generic covid country profile page.
-    const genericCountryProfilePostApi = await wpdb.getPostBySlug(
-        covidCountryProfileSlug
-    )
+const getCountryProfilePost = memoize(
```

Comment:
Nothing to do now but I'm wondering if this should go in a separate file, and how we could refactor Baker a bit. I think it would be nice to ditch the global `settings.ts` as much as possible, to make a more functional codebase. That would also allow us to do easier partial baking and baking things to specific locations.

Reply:
I definitely considered it. The baker needs some organization. This PR touches too many files for my liking. Not surprised there's a memory leak :)